docs/TektonConfig.md

```diff
@@ -248,6 +248,7 @@ Example:
 chain:
   disabled: false                  # - `disabled` : if the value set as `true`, chains feature will be disabled (default: `false`)
   targetNamespace: tekton-pipelines
+  generateSigningSecret: true # default value: false
```

Comment:
Are we handling the scenario where if we change the user change the field value from `true` to `false` `generateSigningSecret` in that case are we deleting the key/values from the secret ?

Reply:
No actually, it doesnt update the signing key. I dont think we want to do so,the property itself generates a signing key, the way operator works with installerset is that
- if generateSigningSecret is true, the installerset is created with the secret data(in the yaml manifest)
- if generateSigningSecret is false, the installerset is created with an empty secret data. Now if the installerset is already there, it is not updated. this is common function for all the component handled by the operator, the user should remove the installerset to recreate an empty one.